react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecManager.java

```diff
@@ -504,6 +513,11 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
                             oldText,
                             start,
                             start + before));
+
+            // Add the outer tags when the field was started empty, and only the first time the user types in it.
+            if (mPreviousText.length() == 0 && currentEventCount == 1 && !TextUtils.isEmpty(mEditText.getTagName())) {
```

Comment:
If you agree we can probably put `mPreviousText.isEmpty()` instad of `mPreviousText.length() == 0`
also it would be to put `currentEventCount == 1` in some constant with proper name.

Reply:
Unfortunately there are cases where Aztec does only contains "non-breaking-space" char and `isEmpty` does return true, but the length is not `0`. This is happening when you write and then remove all the content from the block.

Will address the other comments instead.